source/setup/pc_setup.rst

```diff
@@ -73,4 +73,9 @@ from the host machine to the VM. Follow these instructions in VirtualBox.
         #. Name should be your wired adapter
         
 .. note::
-    In case you're unable to connect to a robot (i.e. `ssh <robotname>` doesn't work), try installing `Wireshark <https://www.wireshark.org/download.html>`_ and monitoring the packets to diagnose the issue. 	Also try pinging by the expected IP address of the robot, `ping <robotname>.local`, and checking the output of `avahi-browse -av`.
+    In case you're unable to connect to a robot (i.e. `ssh <robotname>` doesn't work), try the following steps, proceeding to the next step if the one you tried doesn't work:
+        #. Try `ssh <robotname>.local`.
+        #. Try `ssh nao@<robot_ip_address>`
+        #. If the error message from any of the above steps was something like `Name or service not known`, try `ping <robotname>.local`.
+        #. Try `avahi-browse -av` and check the output to see if the name of the robot you're connecting to is listed.
+        #. Install `Wireshark <https://www.wireshark.org/download.html>`_ and monitor the packets to diagnose the issue. If you find this step overwhelming, please seek help from one of the more experienced members.
```

Comment:
sorry these steps keep changing.  `Name or service not known` implies that your computer can't turn `<robotname>.local` into an ip address.

so, if `ssh nao@<robot ip address>` doesn't work then you need to do some more investigative work.

1. check that your pc is on the network by pinging something else (like the router)
2. check that the robot has the right IP by some chest button combination i can never remember
3. check the router that it thinks the robot is connected (it should appear as a wifi client (wifi) or a dhcp client (wired))
4. try both wifi and wired
5. resort to wireshark (although personally, i would probably reimage the robot)

Reply:
>. check that the robot has the right IP by some chest button combination i can never remember 

4 presses I think according to [Button Presses](https://runswift.readthedocs.io/en/latest/running/button_presses.html#chest-button-interface).

>. check the router that it thinks the robot is connected (it should appear as a wifi client (wifi) or a dhcp client (wired))

So with something like `avahi-browse -av` ?

>. try both wifi and wired
Haven't tried via wifi. So might add this later after I try it out.

Reply:
> check the router

i mean on the router's web interface, not on the PC

Reply:
Ah, okay. But I guess if your PC is connected to the same network, you should be able to see all the devices on the network since they're discoverable by default ?

Reply:
yes, you should be able to see the devices and resolve from name to IP with .local

yeah i do sometimes use `nmap`